docs/concepts/flatcar-container-linux.md

```diff
@@ -0,0 +1,83 @@
+| container runtime      | docker                  |
+| cgroup driver          | cgroupfs                |
+| logging driver         | json-file               |
+| storage driver         | overlay2                |
+
+### Kubernetes properties
+
+| Property                      | Flatcar Container Linux                                                |
+|-------------------------------|------------------------------------------------------------------------|
+| single-master                 | all platforms                                                          |
+| multi-master                  | planned                                                                |
+| control plane                 | self-hosted                                                            |
+| kubelet image                 | upstream hyperkube                                                     |
+| control plane images          | upstream hyperkube                                                     |
+| on-host etcd                  | rkt-fly                                                                |
+| on-host kubelet               | rkt-fly                                                                |
+| CNI plugins                   | calico                                                                 |
+| coordinated drain & OS update | [FLUO](https://github.com/kinvolk/flatcar-linux-update-operator) addon |
+
+### Directory locations
+
+Lokomotive conventional directories:
+
+| Kubelet setting   | Host location                  |
+|-------------------|--------------------------------|
+| cni-conf-dir      | /etc/kubernetes/cni/net.d      |
+| pod-manifest-path | /etc/kubernetes/manifests      |
+| volume-plugin-dir | /var/lib/kubelet/volumeplugins |
+
+### Kubelet mounts
+
+Kubelet mount points on Flatcar Container Linux:
+
+| Mount location             | Host location              | Options   |
+|----------------------------|----------------------------|-----------|
+| /etc/kubernetes            | /etc/kubernetes            | ro        |
+| /etc/ssl/certs             | /etc/ssl/certs             | ro        |
+| /usr/share/ca-certificates | /usr/share/ca-certificates | ro        |
+| /var/lib/kubelet           | /var/lib/kubelet           | recursive |
+| /var/lib/docker            | /var/lib/docker            |           |
+| /var/lib/cni               | /var/lib/cni               |           |
+| /var/lib/calico            | /var/lib/calico            |           |
+| /var/log                   | /var/log                   |           |
+| /etc/os-release            | /usr/lib/os-release        | ro        |
+| /run                       | /run                       |           |
+| /lib/modules               | /lib/modules               | ro        |
+| /etc/resolv.conf           | /etc/resolv.conf           |           |
+| /opt/cni/bin               | /opt/cni/bin               |           |
```

Comment:
This is nice information for the users, but it's Lokomotive who ultimately configures this. Should we move it to somewhere else? Maybe `docs/configuration-reference/distro/host-paths.md`?

Reply:
Its not something the user would be able to configure right ?

So I think configuration-references would not be the best place to put there. Hence I added in the concepts documents, specifically in the flatcar-linux document.

Reply:
@iaguis

Reply:
I don't think it should be in `configuration-reference`, as this docs should be dedicated to user configuration, so I agree with @ipochi here.

Maybe we could have another subdirectory `docs/architecture` and put it there, to collect there all architecture decisions documentations there? Our single-controlplane upgrade concept could also be placed there then.

Reply:
IMO, for now we don't have a lot of documentation related to `architecture`, so I'd say we can temporary place it in the current document. 

Reorder the content once we have more documentation related to architecture ?

Reply:
For me seems fine. @iaguis ?

Reply:
I don't like it too much but I won't push back so all fine for me.

Reply:
I'll mark as resolved then.